news/2024/2024-09-16-the-followpoint-xootynator-the-ceo-of-full-area.md

```diff
@@ -0,0 +1,307 @@
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-lvd" href="https://osu.ppy.sh/users/18152711">MegaMix</a></p>
+
+How do you manage nerves/shake when setting leaderboard scores?
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/3717598" style="background-image: url('/wiki/shared/news/2024-09-16-the-followpoint-xootynator-the-ceo-of-full-area/avatar-xootynator.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-no-group" href="https://osu.ppy.sh/users/3717598">xootynator</a></p>
+
+My best piece of advice for nerves is always this: everyone gets nerves, but what makes you into a really good player is knowing how to play through those nerves. I personally don't shake much but my breathing gets shallow and I notice my mind starts to wander and worry about missing. What keeps me grounded is repeating thoughts like "I could miss, or I could also just FC" or to remind myself to stay focused on the main mechanics of my body. Don't lose your reading and listening, and keep your hands as loose as possible.
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/18152711" style="background-image: url('/wiki/shared/news/2024-09-16-the-followpoint-xootynator-the-ceo-of-full-area/avatar-MegaMix.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-lvd" href="https://osu.ppy.sh/users/18152711">MegaMix</a></p>
+
+What are your thoughts on osu!lazer?
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/3717598" style="background-image: url('/wiki/shared/news/2024-09-16-the-followpoint-xootynator-the-ceo-of-full-area/avatar-xootynator.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-no-group" href="https://osu.ppy.sh/users/3717598">xootynator</a></p>
+
+At first I was really apprehensive, but I've gotten a little more comfortable with osu!lazer now. I still think there are a lot of pros and cons, and the general opinion is pretty mixed. As a competitive player, I think gameplay on Lazer is starting to feel much smoother and more comfortable... only after I spent a couple days shuffling through the frame settings and the different render settings. There's also a lot of missing features, like not being able to know what skin I'm using until I'm in a map, MP links being really weird for tournaments, and not being able to hide the scoreboard when playing multiplayer, which is pretty distracting. I also think Lazer has awesome features and they're always adding cool things, however the UI feels very crowded and I'd have to often ask where things are and how to find certain settings. Totally something you can get used to as a new player fresh in to lazer, but it can be pretty overwhelming for someone like me.
```

Comment:
```suggestion
At first, I was really apprehensive, but I've gotten a little more comfortable with osu!lazer now. I still think there are a lot of pros and cons, and the general opinion is pretty mixed. As a competitive player, I think gameplay on lazer is starting to feel much smoother and more comfortable... only after I spent a couple days shuffling through the frame settings and the different render settings.

There's also a lot of missing features, like not being able to know what skin I'm using until I'm in a map, MP links being really weird for tournaments, and not being able to hide the scoreboard when playing multiplayer, which is pretty distracting. I also think lazer has awesome features and they're always adding cool things, however the UI feels very crowded, and I'd have to often ask where things are and how to find certain settings. Totally something you can get used to as a new player fresh in to lazer, but it can be pretty overwhelming for someone like me.
```